src/synopsys-action/tools-parameter.ts

```diff
@@ -471,6 +456,8 @@ export class SynopsysToolsParameter {
       blackduckData.data.network = {airGap: parseToBoolean(inputs.ENABLE_NETWORK_AIR_GAP)}
     }
 
+    blackduckData.data.blackduck = Object.assign({}, this.setBlackDuckArbitraryArgs(), blackduckData.data.blackduck)
```

Comment:
as we are setting this parameters here, maybe we can remove this at
https://github.com/synopsys-sig/synopsys-action/blob/7f8a3c5341fc8d548989f596ae72e004a958212f/src/synopsys-action/tools-parameter.ts#L380

https://github.com/synopsys-sig/synopsys-action/blob/7f8a3c5341fc8d548989f596ae72e004a958212f/src/synopsys-action/tools-parameter.ts#L386

We are setting same thing twice.

Reply:
Yes